src/test/java/org/mockito/internal/stubbing/defaultanswers/ForwardsInvocationsTest.java

```diff
@@ -0,0 +1,27 @@
+package org.mockito.internal.stubbing.defaultanswers;
+
+import org.junit.Test;
+import org.mockitoutil.TestBase;
+
+public class ForwardsInvocationsTest extends TestBase {
+
+    interface Foo {
+        int bar(String baz, Object... args);
+    }
+
+    static final class FooImpl implements Foo {
+        @Override
+        public int bar(String baz, Object... args) {
+            return baz.length() + args.length;
+        }
+    }
+
+    @Test
+    public void should_call_method_with_varargs() throws Throwable {
+        Foo foo = new FooImpl();
+        ForwardsInvocations forwardsInvocations = new ForwardsInvocations(foo);
+
+        assertEquals(3, forwardsInvocations.answer(invocationOf(Foo.class, "bar", "baz", new Object[] {})));
+        assertEquals(6, forwardsInvocations.answer(invocationOf(Foo.class, "bar", "baz", new Object[] {12, "34", 5.67})));
```

Comment:
- Looks good, can you split the test cases (empty array/ non empty array) ?
- Can you also add a test for `invocationOf(Foo.class, "bar", "baz", new Object[] {null})` and `invocationOf(Foo.class, "bar", "baz", null)` 
- Would you mind using AssertJ's Assertions.assertThat(..).is(..), it is more readable IMHO?